lnbits/wallets/cln.py

```diff
@@ -23,23 +22,9 @@
 )
 
 
-def async_wrap(func):
-    @wraps(func)
-    async def run(*args, loop=None, executor=None, **kwargs):
-        if loop is None:
-            loop = asyncio.get_event_loop()
-        partial_func = partial(func, *args, **kwargs)
-        return await loop.run_in_executor(executor, partial_func)
-
-    return run
-
-
-def _pay_invoice(ln, payload):
-    return ln.call("pay", payload)
-
-
-def _paid_invoices_stream(ln, last_pay_index):
-    return ln.waitanyinvoice(last_pay_index)
+async def run_sync(func) -> Any:
```

Comment:
- how is this related to `asyncio.create_task` improvements?

Reply:
https://github.com/lnbits/lnbits/pull/1858

Reply:
this fix was required to get tests to pass. cln would otherwise get stuck